config/initializers/sphinx.rb

```diff
@@ -8,3 +8,15 @@ def indices_of_type(type)
     super.reject(&method(:inheritance_columns?))
   end
 end)
+
+# workaround unresolved issue with manticore total_pages
+# fixes SearchPresentersTest#test_pagination
+# see https://github.com/pat/thinking-sphinx/pull/1213
+ThinkingSphinx::Masks::PaginationMask.prepend(Module.new do
+  def total_pages
+    return 0 unless search.meta['total_found']
```

Comment:
So, would this also work for Sphinx? From what I read:
> When we do the pagination we should consider the `total`, because if we use `total_found` we might try to display more pages than we can actually show.

Or is `[total_entries, 1000].min` protecting from it? (i.e. we will never try to show results that are after 1000's entry, so we are safe)

Also, the methods are confusing in TS, it's not very clear that `#total_entries` refer to the same number as `search.meta['total_found']`, as well as `#total_count` and `#count` :sweat_smile: But that's OK.

Reply:
See description. It should work alike on manticore and sphinx.

Reply:
yes

> `[total_entries, 1000].min` protecting from it